scripts/install.sh

```diff
@@ -175,14 +173,26 @@ function outro() {
     KUBEADM_TOKEN_CA_HASH=$(cat /tmp/kubeadm-init | grep 'discovery-token-ca-cert-hash' | awk '{ print $2 }' | head -1)
 
     printf "\n"
-    printf "\t\t${GREEN}Installation${NC}\n"
-    printf "\t\t${GREEN}  Complete ✔${NC}\n"
+    printf "\t\t${green}installation${nc}\n"
```

Comment:
Did you mean to lowercase lines 176-182?

Reply:
fixed